prow/lib.sh

```diff
@@ -113,8 +113,8 @@ function build_images() {
   # Build just the images needed for tests
   targets="docker.pilot docker.proxyv2 "
 
-  # use ubuntu:jammy to test vms by default
-  nonDistrolessTargets="docker.app docker.app_sidecar_ubuntu_jammy docker.ext-authz "
+  # use centos:8 to test vms by default
+  nonDistrolessTargets="docker.app docker.app_sidecar_rockylinux_8 docker.ext-authz "
```

Comment:
I am not sure what to do with this change, because the original commit was changing Ubuntu Jammy to CentOS 8, but now upstream uses Rocky Linux instead of CentOS 8. What's more, tests on Rocky Linux (which is supposed to be an equivalent of RHEL/CentOS) don't work and were disabled. So since we don't support VMs yet, I suggest to skip this change for now and investigate this problem once we are ready to support VMs.
WDYT @dgn @jwendell?